<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MathW</class>
 <widget class="QWidget" name="MathW">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>329</width>
    <height>293</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>329</width>
    <height>293</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>311</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="copy">
      <property name="text">
       <string>复制</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="paste">
      <property name="text">
       <string>粘贴</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="clear">
      <property name="text">
       <string>清空</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="back">
      <property name="text">
       <string>后退</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="enter">
      <property name="text">
       <string>确认</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>201</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="accX">
        <property name="text">
         <string>accX</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="accY">
        <property name="text">
         <string>accY</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="accZ">
        <property name="text">
         <string>accZ</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Acc">
        <property name="text">
         <string>Acc</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="gyroX">
        <property name="text">
         <string>gyroX</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="gyroY">
        <property name="text">
         <string>gyroY</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="gyroZ">
        <property name="text">
         <string>gyroZ</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Gyro">
        <property name="text">
         <string>Gyro</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QPushButton" name="magX">
        <property name="text">
         <string>magX</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="magY">
        <property name="text">
         <string>magY</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="magZ">
        <property name="text">
         <string>magZ</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Mag">
        <property name="text">
         <string>Mag</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QPushButton" name="angleX">
        <property name="text">
         <string>angleX</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="angleY">
        <property name="text">
         <string>angleY</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="angleZ">
        <property name="text">
         <string>angleZ</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Angle">
        <property name="text">
         <string>Angle</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>211</x>
     <y>110</y>
     <width>111</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="one">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="two">
        <property name="text">
         <string>2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="three">
        <property name="text">
         <string>3</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="add">
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="four">
        <property name="text">
         <string>4</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="five">
        <property name="text">
         <string>5</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="six">
        <property name="text">
         <string>6</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="sub">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="seven">
        <property name="text">
         <string>7</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="eight">
        <property name="text">
         <string>8</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="nine">
        <property name="text">
         <string>9</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="mul">
        <property name="text">
         <string>*</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QPushButton" name="left">
        <property name="text">
         <string>(</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="zero">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="right">
        <property name="text">
         <string>)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="div">
        <property name="text">
         <string>/</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
